site/content/en/docs/manual/advanced/annotation-with-skeletons/auto-skeletons.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Automatic annotation with Skeletons'
+linkTitle: 'Automatic annotation with Skeletons'
+weight: 2
+description: 'Automatic annotation with Skeletons'
+---
+
+In this guide, we delve into the efficient process of annotating complex
+structures through the implementation of **Skeleton** automatic annotations.
+
+See:
+
+- [Creating auto annotation Skeleton task](#creating-auto-annotation-skeleton-task)
+- [Automatic annotation with Skeletons](#automatic-annotation-with-skeletons)
+  - [Editing skeletons on the canvas](#editing-skeletons-on-the-canvas)
+
+## Creating auto annotation Skeleton task
+
+To auto annotation **Skeleton** task, do the following:
+
+1. Open **Basic configurator**.
+2. On the **Constructor** tab, click **From model**.
+3. From the **Select a model to pick labels** select the model
+   you want to use.
+4. Click on the model's labels, you want to use.
+   Selected labels will become gray.
+
+   ![](/images/auto-annot-sk.jpg)
+
+5. Click **Done**. Labels, that you selected,
+   will appear in the labels window.
+6. (Optional) If you want to adjust labels, within the
+   label, click the **Update** attributes icon.
+   <br>The [Skeleton configurator](/docs/manual/advanced/annotation-with-skeletons/manual-skeletons/#skeleton-task)
+   will open, where you can [configure the skeleton](/docs/manual/advanced/annotation-with-skeletons/manual-skeletons/#configuring-skeleton-points).
+7. Upload data.
+8. Click:
+   - **Submit & Open** to create and open the task.
+   - **Submit & Continue** to submit the configuration and start creating a new task.
+
+## Automatic annotation with Skeletons
+
+To automatically annotate with **Skeleton**, do the following
+
+1. Open job and on the tools panel select **AI Tools** > **Detectors**
+2. From the drop-down list select model.
+   You will see list of points to match and
+   the name of the label on the top of the list.
```

Comment:
```suggestion
   You will see list of points to match and
   the name of the skeleton on the top of the list.
```

Reply:
That is not actually true. 

1. User will see mapping configurator to map model skeletons to task skeletons. For each mapped skeleton user will see mapping configurator for points of paired skeletons.
2. But those skeletons that have the same name in the model, and in the task, will be mapped automatically. In points mapping configurator, points from paired skeletons with the same name, also will be mapped automatically.